src/app/pages/applications/chart-releases/chart-releases.component.ts

```diff
@@ -52,7 +53,7 @@ export class ChartReleasesComponent implements AfterViewInit, OnInit, OnDestroy
   filteredChartItems: ChartRelease[] = [];
   filterString = '';
 
-  chartItems: Record<string, ChartRelease> = {};
+  chartItems = new Map<string, ChartRelease>();
   @Output() switchTab = new EventEmitter<string>();
 
   private dialogRef: MatDialogRef<EntityJobComponent>;
```

Comment:
can we refactor this .ts file to remove **this.dialogRef** and use const **dialogRef** instead?